### ISSUE TITLE
Scoped strict mode for AMD and UMD modules

### ISSUE THREAD

**Issue by morrisallison** [Microsoft/TypeScript#5288]
Putting the following code into the Playground:

```typescript
"use strict";
var foo = 'foo';
export var bar = 'bar';
```

generates this:

```javascript
"use strict";
define(["require", "exports"], function (require, exports) {
    var foo = 'foo';
    exports.bar = 'bar';
});
```

When I expected to see this:

```javascript
define(["require", "exports"], function (require, exports) {
    "use strict";
    var foo = 'foo';
    exports.bar = 'bar';
});
```

I'm wondering why `"use strict";` exists outside of the module.

**Comment by weswigham** [comment]
This should be fixed in master: #5116

**Comment by vladima** [comment]
@weswigham is right, latest bits from master emit
```ts
define(["require", "exports"], function (require, exports) {
    "use strict";
    var foo = 'foo';
    exports.bar = 'bar';
});

```
